frontend/src/app/pages/exercises/exercise/shared/simulated-region-overview/radiogram-list/radiogram-card/radigoram-card-content-resource-request/radigoram-card-content-resource-request.component.html

```diff
@@ -14,31 +17,54 @@ <h5>Anfrage von Ressourcen</h5>
             <tbody>
                 <tr
                     *ngFor="
-                        let vehicleType of requiredResource.vehicleCounts | keys
+                        let vehicleType of radiogram.requiredResource
+                            .vehicleCounts | keys
                     "
                 >
                     <td>{{ vehicleType }}</td>
-                    <td>{{ requiredResource.vehicleCounts[vehicleType] }}</td>
+                    <td>
+                        {{
+                            radiogram.requiredResource.vehicleCounts[
+                                vehicleType
+                            ]
+                        }}
+                    </td>
                 </tr>
             </tbody>
         </table>
         <div class="d-flex row my-1">
-            <button
-                type="button"
-                [disabled]="(enableActionButtons$ | async) === false"
-                (click)="denyRequest()"
-                class="btn btn-danger col-sm mx-1"
+            <ng-container
+                *ngIf="
+                    radiogram.resourcesPromised === undefined;
+                    else requestAnswered
+                "
             >
```

Comment:
If the radiogram is in done status, the buttons should not be displayed even if the success of the request is unknown. I would suggest to show a text that the success in unknown instead.